packages/office-ui-fabric-react/src/components/TextField/examples/NumberTextField.tsx

```diff
@@ -41,6 +44,12 @@ export class NumberTextField extends React.Component<INumberTextFieldProps, INum
     );
   }
 
+  private _updateRef = (ref: any) => {
```

Comment:
Can you just use createRef instead? Probably a better pattern to demonstrate.

Reply:
We should also be able to type the `ref`

Reply:
Good call.